tests/disruption_helpers.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from resources import pod
+from ocs import constants, ocp
+from ocsci import config
+logger = logging.getLogger(__name__)
+
+POD = ocp.OCP(kind=constants.POD, namespace=config.ENV_DATA['cluster_namespace'])
+
+
+class Disruptions:
+    """
+    TBD
+    """
+    def get_mgr(self):
+        self.mgr_pod = pod.get_ceph_mgr_pod()
+
+    def get_mon(self):
+        self.mons = pod.get_ceph_mon_pods()
+
+    def get_mds(self):
+        self.mds_pods = pod.get_ceph_mon_pods()
+
+    def get_osd(self):
+        self.osd_pods = pod.get_ceph_mon_pods()
+
+    def delete_mgr(self):
+        self.mgr_pod.delete()
+        assert POD.wait_for_resource(condition='Running', selector='app=rook-ceph-mgr')
+
+    def delete_mon(self):
+        for mon in self.mons:
+            mon.delete()
+        assert POD.wait_for_resource(condition='Running', selector='app=rook-ceph-mon')
```

Comment:
@ebenahar are we planning to respin all the mon pods almost one after the other ? Without any sleep ?

JFYI, we need two MONs to be up for quorum to be met and ceph commands to run.

Reply:
@nehaberry how much time do you think we can sleep between MONs deletions?